scrapi/harvesters/crossref.py

```diff
@@ -26,13 +26,12 @@
 def process_contributor(author, orcid):
     name = HumanName(author)
     return {
-        'prefix': name.title,
-        'given': name.first,
-        'middle': name.middle,
-        'family': name.last,
-        'suffix': name.suffix,
+        'name': author,
```

Comment:
process_contributors is used in a lot of places, but I think this is the only harvester that processes one contributor at a time.